# SOME DESCRIPTIVE TITLE.
# Copyright (C) 2001-2025, Python Software Foundation
# This file is distributed under the same license as the Python package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
# Translators:
# eulalio barbero espinosa <eulalio@disroot.org>, 2023
# Rafael Fontenelle <rffontenelle@gmail.com>, 2025
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: Python 3.10\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2025-05-02 16:07+0000\n"
"PO-Revision-Date: 2022-11-05 17:21+0000\n"
"Last-Translator: Rafael Fontenelle <rffontenelle@gmail.com>, 2025\n"
"Language-Team: Spanish (Spain) (https://app.transifex.com/python-doc/"
"teams/5390/es_ES/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: es_ES\n"
"Plural-Forms: nplurals=3; plural=n == 1 ? 0 : n != 0 && n % 1000000 == 0 ? "
"1 : 2;\n"

#: ../../library/asyncio-exceptions.rst:8
msgid "Exceptions"
msgstr "Excepciones"

#: ../../library/asyncio-exceptions.rst:10
msgid "**Source code:** :source:`Lib/asyncio/exceptions.py`"
msgstr "**Código fuente:** :source:`Lib/asyncio/exceptions.py`"

#: ../../library/asyncio-exceptions.rst:16
msgid "The operation has exceeded the given deadline."
msgstr ""

#: ../../library/asyncio-exceptions.rst:19
msgid ""
"This exception is different from the builtin :exc:`TimeoutError` exception."
msgstr ""

#: ../../library/asyncio-exceptions.rst:25
msgid "The operation has been cancelled."
msgstr "Se ha cancelado la operación."

#: ../../library/asyncio-exceptions.rst:27
msgid ""
"This exception can be caught to perform custom operations when asyncio Tasks "
"are cancelled.  In almost all situations the exception must be re-raised."
msgstr ""
"Esta excepción se puede capturar para realizar operaciones personalizadas "
"cuando se cancelan Tareas asyncio.  En casi todas las situaciones la "
"excepción se debe re-lanzar."

#: ../../library/asyncio-exceptions.rst:33
msgid ":exc:`CancelledError` is now a subclass of :class:`BaseException`."
msgstr ""

#: ../../library/asyncio-exceptions.rst:38
msgid "Invalid internal state of :class:`Task` or :class:`Future`."
msgstr "Estado interno no válido de :class:`Task` o :class:`Future`."

#: ../../library/asyncio-exceptions.rst:40
msgid ""
"Can be raised in situations like setting a result value for a *Future* "
"object that already has a result value set."
msgstr ""
"Puede aparecer en situaciones como establecer un valor de resultado para un "
"objeto *Future* que ya tiene un valor de resultado establecido."

#: ../../library/asyncio-exceptions.rst:46
msgid ""
"The \"sendfile\" syscall is not available for the given socket or file type."
msgstr ""
"La llamada al sistema \"sendfile\" no está disponible para el socket o tipo "
"de archivo dados."

#: ../../library/asyncio-exceptions.rst:49
msgid "A subclass of :exc:`RuntimeError`."
msgstr "Una subclase de :exc:`RuntimeError`."

#: ../../library/asyncio-exceptions.rst:54
msgid "The requested read operation did not complete fully."
msgstr "La operación de lectura solicitada no se ha completado."

#: ../../library/asyncio-exceptions.rst:56
msgid "Raised by the :ref:`asyncio stream APIs<asyncio-streams>`."
msgstr "Activado por las :ref:`asyncio stream APIs<asyncio-streams>`."

#: ../../library/asyncio-exceptions.rst:58
msgid "This exception is a subclass of :exc:`EOFError`."
msgstr "Esta excepción es una subclase de :exc:`EOFError`."

#: ../../library/asyncio-exceptions.rst:62
msgid "The total number (:class:`int`) of expected bytes."
msgstr "El número total (:class:`int`) de bytes esperados."

#: ../../library/asyncio-exceptions.rst:66
msgid "A string of :class:`bytes` read before the end of stream was reached."
msgstr "Una cadena de :class:`bytes` leída antes de llegar al final del flujo."

#: ../../library/asyncio-exceptions.rst:71
msgid "Reached the buffer size limit while looking for a separator."
msgstr ""
"Alcanzado el límite de tamaño del búfer mientras se buscaba un separador."

#: ../../library/asyncio-exceptions.rst:73
msgid "Raised by the :ref:`asyncio stream APIs <asyncio-streams>`."
msgstr "Activado por las :ref:`asyncio stream APIs <asyncio-streams>`."

#: ../../library/asyncio-exceptions.rst:77
msgid "The total number of to be consumed bytes."
msgstr "El número total de bytes a consumir."
